test/clientPolicies.spec.ts

```diff
@@ -1,21 +1,43 @@
 // tslint:disable:no-unused-expression
 import * as chai from 'chai';
 import {KeycloakAdminClient} from '../src/client';
+import ClientProfilesRepresentation from '../src/defs/clientProfilesRepresentation';
 import {credentials} from './constants';
 
 const expect = chai.expect;
 
 describe('Client Policies', () => {
   let kcAdminClient: KeycloakAdminClient;
+  let currentClientProfiles: ClientProfilesRepresentation;
```

Comment:
Tests cases should not have dependencies between each other as this can cause failures if they are ran individually.

Reply:
thanks @jonkoops - you're correct!